webapp/tests/forms/flows/test_lotse_flow.py

```diff
@@ -984,21 +984,23 @@ def test_if_step_is_declaration_edaten_step_and_non_valid_form_then_do_not_call_
 
                 create_audit_log_fun.assert_not_called()
 
-    def test_if_familienstand_step_then_delete_person_b_data_correctly(self):
```

Comment:
There is one line in this file still referencing `is_person_a_account_holder` in l. 1530.